acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/TransactionSignerParamsSupplier.java

```diff
@@ -28,34 +28,55 @@
 import com.google.common.io.Resources;
 
 public class TransactionSignerParamsSupplier {
+
   private final int hashicorpVaultPort;
   private final String ipAddress;
+  private final String azureKeyVault;
 
-  public TransactionSignerParamsSupplier(final int hashicorpVaultPort, final String ipAddress) {
+  public TransactionSignerParamsSupplier(
+      final int hashicorpVaultPort, final String ipAddress, final String azureKeyVault) {
     this.hashicorpVaultPort = hashicorpVaultPort;
     this.ipAddress = ipAddress;
+    this.azureKeyVault = azureKeyVault;
   }
 
   public Collection<String> get() {
     final ArrayList<String> params = new ArrayList<>();
-    if (hashicorpVaultPort == 0) {
-      params.add("file-based-signer");
-      params.add("--password-file");
-      params.add(createPasswordFile().getAbsolutePath());
-      params.add("--key-file");
-      params.add(createKeyFile().getAbsolutePath());
-    } else {
+    if (hashicorpVaultPort != 0) {
       params.add("hashicorp-signer");
       params.add("--auth-file");
       params.add(createVaultAuthFile().getAbsolutePath());
       params.add("--host");
       params.add(ipAddress);
       params.add("--port");
       params.add(String.valueOf(hashicorpVaultPort));
+    } else if (azureKeyVault != null) {
+      params.add("azure-signer");
+      params.add("--keyvault-name");
+      params.add(azureKeyVault);
+      params.add("--key-name");
+      params.add("TestKey");
+      params.add("--key-version");
+      params.add("7c01fe58d68148bba5824ce418241092");
+      params.add("--client-id");
+      params.add(System.getenv("ETHSIGNER_AZURE_CLIENT_ID"));
```

Comment:
thought you were getting rid of the system environment variables?

Reply:
We are going to need to keep them for when we do go to a Jenkins-credential system (I think) so left them here.